### DISCUSSION TITLE
Update-PnPTerm not working

### DISCUSSION THREAD

**Discussion by antso77** [pnp/powershell#366] [General] [answered]
I'm confused: what is the correct way to use Set-PnPTerm? Am I missing some module?
None of these work Set-PnPTerm OR Update-PnPTerm

Examples

COMMAND: Set-PnPTerm -Identity "3d9e60e8-d89c-4cd4-af61-a010cf93b380" -Name "New Name"

ERROR: Set-PnPTerm : Object reference not set to an instance of an object.
At line:1 char:1
+ Set-PnPTerm -Identity 3d9e60e8-d89c-4cd4-af61-a010cf93b380 -Name "New ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPTerm], NullReferenceException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Taxonomy.SetTerm

COMMAND: Update-TermSet -Identity "Marketing" -TermSet "Departments" -TermGroup "Corporate" -Name "NewName"

ERROR: Update-TermSet : The term 'Update-TermSet' is not recognized as the name of a cmdlet, function, script file, or operabl
e program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ Update-TermSet -Identity "Marketing" -TermSet "Departments" -TermGrou ...
+ ~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Update-TermSet:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by veronicageek** [comment, marked as answer]
Hi @antso77 - Indeed, there's an error in the documentation, I'll fix that.
The command `Set-PnPTerm` works correctly though. (FYI - There's no cmdlet called Update-PnPTerm)

![image](https://user-images.githubusercontent.com/25181757/110221009-7e702a00-7ec1-11eb-820d-6d92fe004940.png)
